flink-cdc-common/src/main/java/com/ververica/cdc/common/event/DataChangeEvent.java

```diff
@@ -131,4 +131,30 @@ public static DataChangeEvent replaceEvent(
             TableId tableId, RecordData after, Map<String, String> meta) {
         return new DataChangeEvent(tableId, null, after, OperationType.REPLACE, meta);
     }
+
+    /** Creates a string to describe the information of meta. */
+    public String describeMeta() {
+        StringBuilder stringBuilder = new StringBuilder("(");
+        if (meta != null && !meta.isEmpty()) {
+            stringBuilder.append(meta);
+        }
+        stringBuilder.append(")");
+        return stringBuilder.toString();
+    }
+
+    @Override
+    public String toString() {
```

Comment:
I think we can wait the [PR](https://github.com/ververica/flink-cdc-connectors/pull/2714) merged and then rebase it